HW03/Alexey_Zalyotov/lib/mentor.rb

```diff
@@ -0,0 +1,42 @@
+class Mentor < Person
+  def create_repo
+    Repo.new
+  end
+
+  def create_task!(task_body, repo)
+    @task_in_progress = Homework.new(task: task_body, repo: repo)
+    repo.homeworks << task_in_progress
+    repo.mentors << self
+    notificate_students(task_in_progress)
+  end
+
+  def review_task!(comment)
+    task_in_progress.status = :reviewed
+    task_in_progress.review_comment = comment
+    notificate_students(task_in_progress)
+  end
+
+  def approve_homework!
```

Comment:
be consistent, use only one name everywhere `homework or task'
```suggestion
  def approve_task!
```

Reply:
Done